app/views/sites/api_instructions/_show_web_md.html.haml

```diff
@@ -1,485 +1,77 @@
 :markdown
-  <!-- Reminder to update https://open.gsa.gov/api/searchgov-results/ with any changes here. -->
-
-  This API exposes all relevant results “modules” in a single JSON call, including:
-
-
+  ## Overview
 
+  The Search.gov Results API exposes all relevant results “modules” in a single JSON call, including the following:
```

Comment:
Maybe just a nit, but I'm not a fan of emphasis quotes. What are you trying to convey with them, and is there a better way to do it? Maybe a link somewhere, if modules is a term with specific meaning you want to emphasize here, or just _underline_ or *star* emhasis.